src/sites/amsterdam/components/contact/Contact.tsx

```diff
@@ -0,0 +1,64 @@
+import { Breadcrumb, Column, Fieldset, Grid, Heading, Link, Paragraph, Radio } from '@amsterdam/design-system-react'
+import { siteUrl } from '../../constants'
+import FormNavigation from '../shared/FormNavigation'
+import { FormEvent } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+export const Contact = () => {
+  const navigate = useNavigate()
+
+  const handleSubmit = async (event: FormEvent<HTMLFormElement>) => {
```

Comment:
Without checking the ‘Vraag’ radio, the ‘Volgende’ button does nothing. Is that intentional because we don’t have an error text yet?